### ISSUE TITLE
Enemy aggro to local area Host

### ISSUE THREAD

**Issue by Capcom177** [tiltedphoques/bug-feature-reports#16]
Initially thought to be aggro all towards host over other players, I have tested with friends and it seems that a player is chosen within a set area as the main target. Any players going into the area after that player has become the target will receive little-to-no enemy aggro.

This has resulted in myself running away from enemies as they ignore the other players in the game and it is very difficult to pull the aggression off to another target.

I don't believe this is a host issue as I had left the host in a completely different area to test. Myself and another player went into the area and he was pulling almost 100% of the aggro that we encountered and I could not get targets to switch to me.

**Comment by Wehrmicel** [comment]
Yep I noticed the same thing with my friend. Enemies are either ignoring the 2nd player or quicly swap their target multiple time.

**Comment by GreenMachine2o3** [comment]
This is exactly what happened to me. Honestly it made it virtually unplayable, one of my friends would just run around with full aggro while myself and another friend attacked the enemies as they chased him and we never pulled any aggro

**Comment by tmanochehri** [comment]
It was only when my friend would do enough damage that the enemies would attack her. Most of the time they aggro on me.

**Comment by LucasRoquilly** [comment]
Yes, I think this is the biggest issue to fix, with random events/actors not syncing.
